config/initializers/menus.rb

```diff
@@ -201,112 +214,131 @@
   menu.push :ldap_authentication,
             { controller: '/ldap_auth_sources', action: 'index' },
+            if: Proc.new { User.current.admin? },
             parent: :authentication,
             html: { class: 'server_authentication' },
             last: true,
             if: proc { !OpenProject::Configuration.disable_password_login? }
 
   menu.push :oauth_applications,
             { controller: '/oauth/applications', action: 'index' },
+            if: Proc.new { User.current.admin? },
             parent: :authentication,
             caption: :'oauth.application.plural',
             html: { class: 'oauth_applications' }
 
   menu.push :announcements,
             { controller: '/announcements', action: 'edit' },
+            if: Proc.new { User.current.admin? },
             caption: :label_announcement,
             icon: 'icon2 icon-news'
 
   menu.push :plugins,
             { controller: '/admin', action: 'plugins' },
+            if: Proc.new { User.current.admin? },
             last: true,
             icon: 'icon2 icon-plugins'
 
   menu.push :info,
             { controller: '/admin', action: 'info' },
+            if: Proc.new { User.current.admin? },
             caption: :label_information_plural,
             last: true,
             icon: 'icon2 icon-info1'
 
   menu.push :custom_style,
             { controller: '/custom_styles', action: 'show' },
+            if: Proc.new { User.current.admin? },
             caption: :label_custom_style,
             icon: 'icon2 icon-design'
 
   menu.push :colors,
             { controller: '/colors', action: 'index' },
+            if: Proc.new { User.current.admin? },
             caption: :'timelines.admin_menu.colors',
             icon: 'icon2 icon-status'
 
   menu.push :enterprise,
             { controller: '/enterprises', action: 'show' },
             caption: :label_enterprise_edition,
             icon: 'icon2 icon-headset',
-            if: proc { OpenProject::Configuration.ee_manager_visible? }
+            if: proc { User.current.admin? && OpenProject::Configuration.ee_manager_visible? }
 
   menu.push :admin_costs,
             { controller: '/settings', action: 'plugin', id: :costs },
+            if: Proc.new { User.current.admin? },
             caption: :project_module_costs,
             icon: 'icon2 icon-budget'
 
   menu.push :costs_setting,
             { controller: '/settings', action: 'plugin', id: :costs },
+            if: Proc.new { User.current.admin? },
             caption: :label_setting_plural,
             parent: :admin_costs
 
   menu.push :admin_backlogs,
             { controller: '/settings', action: 'plugin', id: :openproject_backlogs },
+            if: Proc.new { User.current.admin? },
             caption: :label_backlogs,
             icon: 'icon2 icon-backlogs'
 
   menu.push :backlogs_settings,
             { controller: '/settings', action: 'plugin', id: :openproject_backlogs },
+            if: Proc.new { User.current.admin? },
```

Comment:
It would be nice if defaults for the menu items could be provided so that the repetition in here could be avoided.